RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+[!!Documents](RetailSuiteDocorg)  
+[!!Einkauf](RetailSuiteSourcing)   
+[!!Zahlungsabwicklung](RetailSuiteBanking)  
+[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
+[!!User Interface book receipts](../UserInterface/01f_BookReceipts.md)  
+[!!User Interface supplier receipts](../UserInterface/01_Book.md#supplier-receipts)  
+[!!User Interface debtors/creditors](../UserInterface/02a_DebtorsCreditors.md)    
+
+
+# Manage receipts
+
+Receipts are used to document all transactions and, therefore, play an indispensable role for a correct accounting. There are many different types of receipts, such as invoice, order confirmation, or credit note, among others.
+
+The *Accounting* module offers three possible ways to manage and assign receipts to bookings:
+
+- Clicking the [ASSIGN RECEIPT] button in the *BOOKINGS* tab to assign a receipt to an already existing booking in the bookings list; see [Retrieve a receipt](#retrieve-a-receipt).
+- Clicking the *BOOK RECEIPTS* tab to assign a receipt available as a PDF and create a booking manually; see [Book a receipt](#book-a-receipt).
+- Clicking the *Supplier receipts* menu entry in the *EDIT* context menu to specifically assign a supplier receipt; see [Book a supplier receipt](#book-a-supplier-receipt)
+
+> [Info] *Supplier receipts* is a special feature that interacts with the *Purchasing* and *Payment processing* modules, and therefore the appropriate settings must be configured first.
+
+
+When a receipt is booked in the *BOOK RECEIPTS* tab, it is linked to the assigned booking. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon will be then displayed next to the booking in the bookings list. When clicking the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon, the corresponding receipt is displayed.
```

Comment:
Ich glaube die Info kannst du hier löschen - die erwähnst du genau richtig an den entsprechenden Stellen in der Procedure.